api/_mixins/DocumentOrShadowRoot__Document.json

```diff
@@ -239,15 +193,15 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
         }
       },
       "elementsFromPoint": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentOrShadowRoot/elementsFromPoint",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/elementsFromPoint",
```

Comment:
The Edge entry needs updating here. `msElementsFromPoint` was removed in 79, and the unprefixed variant added instead.